go.mod

```diff
@@ -4,5 +4,6 @@ go 1.13
 
 require (
 	github.com/pkg/errors v0.8.1
+	github.com/willf/bitset v1.1.10
```

Comment:
This dependency is BSD-3 Licensed; I _think_ that's ok, but thought I'd add a comment for visibility https://github.com/willf/bitset